pkg/cmd/update-tls-artifacts/generate-owners/tlsmetadatainterfaces/helpers.go

```diff
@@ -22,6 +22,19 @@ func AnnotationValue(whitelistedAnnotations []certgraphapi.AnnotationValue, key
 	return "", false
 }
 
+func DescriptionFor(in []certgraphapi.AnnotationValue) string {
+	ret, _ := AnnotationValue(in, annotations.OpenShiftDescription)
+	return ret
+}
+
+func OwnerFor(in []certgraphapi.AnnotationValue) string {
```

Comment:
That seems to treat "Unknown" as string, so markdown report now doesn't have those listed first, which is a handy feature.

Imo it should return `(string, bool)` and have those listed separate from violations